packages/core/admin/ee/server/services/review-workflows/stages.js

```diff
@@ -1,36 +1,88 @@
 'use strict';
 
+const { mapAsync } = require('@strapi/utils');
+
 const { STAGE_MODEL_UID } = require('../../constants/workflows');
+const { getService } = require('../../utils');
+
+module.exports = ({ strapi }) => {
+  const workflowsService = getService('workflows', { strapi });
+
+  return {
+    find({ workflowId, populate }) {
+      const params = {
+        filters: { workflow: workflowId },
+        populate,
+      };
+      return strapi.entityService.findMany(STAGE_MODEL_UID, params);
+    },
+
+    findById(id, { workflowId, populate }) {
+      const params = {
+        filters: { workflow: workflowId },
+        populate,
+      };
+      return strapi.entityService.findOne(STAGE_MODEL_UID, id, params);
+    },
+
+    createMany(stagesList, { fields }) {
+      const params = {
+        select: fields,
+      };
+      return Promise.all(
+        stagesList.map((stage) =>
+          strapi.entityService.create(STAGE_MODEL_UID, { data: stage, ...params })
```

Comment:
surprised to see there is no createMany in the entityService :/ But there is in the db layer. Do we know why that is?

Reply:
I don't know. But since there will be few stages to create at once, I don't think we need a createMany for this case.